src/plugins/intel_gpu/src/kernel_selector/cl_kernels/rope_ref.cl

```diff
@@ -28,14 +28,11 @@ KERNEL(rope_ref)(
     uint r = rf < HALF_ROTARY_NDIMS ? rf * 2 : 0;
     uint f = rf < HEAD_SIZE - ROTARY_NDIMS ? rf * 2 : 0;
 
-#ifdef ENABLE_SLICE
-    uint input_idx = GET_DATA_INDEX(SLICED_INPUT0, p, b, h * HEAD_SIZE, 0);
-
-    input_idx += SLICED_FROM_START * (p * INPUT0_FEATURE_NUM + b + 1)
-              + SLICED_FROM_END * (p * INPUT0_FEATURE_NUM + b);
-#else
     uint input_idx = INPUT0_GET_INDEX(p, b, h * HEAD_SIZE, 0);
+#ifdef ENABLE_SLICE
```

Comment:
If the new method of calculating indices is correct, then the constant definitions from rope_kernel_base.cpp should be removed too

Reply:
Removed, thanks